front-end/src/app/team-service/team.service.ts

```diff
@@ -0,0 +1,30 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Team } from '../types/Team';
+import { Player } from '../types/Player';
+import { Club } from '../types/Club';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class TeamService {
+
+  host = 'http://localhost:8080';
+
+  constructor(private http: HttpClient) { }
+
+  public getTeam(teamId : string): Observable<Team> {
+    return this.http.get<Team>(`${this.host}/teams/${teamId}`);
+  }
+
+  public getClubOfTeam(clubId : string) : Observable<Club> {
+    return this.http.get<Club>(`${this.host}/teams/${clubId}`);
+  }
+
+  public getPlayersInTeam(teamId: string): Observable<Player[]> {
+    // Endpoint pending. Show all players for now.
```

Comment:
We don't have a nice endpoint yet. We could filter the player list on frontend. For now, we just show all players on every team.

Reply:
Good shout, just did.

#35